Source/Xamarin/Prism.Forms/Navigation/IMultiPageNavigationAwareAsync.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
1) I have this running right now and the Async works better for my needs than the non-async version.
2) Ultimately it's up to the developer to make choices when working with async code that provide for the best user experience.